packages/js/src/plugins/auctionHouseModule/operations/createListing.ts

```diff
@@ -407,7 +407,6 @@ export const createListingBuilder = (
     key.pubkey.equals(signer.publicKey)
   );
   sellInstruction.keys[signerKeyIndex].isSigner = true;
-  sellInstruction.keys[signerKeyIndex].isWritable = true;
```

Comment:
Can you leave this line and mark it as `false`? This attribute should not be undefined.

Reply:
It's not undefined, the line above is to modify instruction keys to align it with provided signers

Reply:
Oh got it! Sorry about that.